### ISSUE TITLE
Bug: Lexer.matchAtrulePrelude Cannot read properties of null (reading 'match')

### ISSUE THREAD

**Issue by carlosjeurissen** [csstree/csstree#203]
When having at-rules like @font-face, I'm getting the following error since v2.2.0:
```
TypeError: Cannot read properties of null (reading 'match')
    at matchSyntax (/node_modules/.pnpm/css-tree@2.2.0/node_modules/css-tree/cjs/lexer/Lexer.cjs:82:51)
    at Lexer.matchAtrulePrelude (/node_modules/.pnpm/css-tree@2.2.0/node_modules/css-tree/cjs/lexer/Lexer.cjs:289:16)
```

**Comment by lahmatiy** [comment]
Fixed in [`v2.2.1`](https://github.com/csstree/csstree/releases/tag/v2.2.1)

**Comment by carlosjeurissen** [comment]
Thank you @lahmatiy !
